src/utils.py

```diff
@@ -1,8 +1,18 @@
-def pretty_print(dictionary):
-    for key, value in dictionary.items():
-        print(f"{key}:")
-        if(type(value) == dict or type(value) == list):
-            pretty_print(value)
-        else:
-            print(value)
-        print()
\ No newline at end of file
+import json
+
+
+def pretty_print(data, indent=0):
+    if isinstance(data, dict):
+        for key, value in data.items():
+            print(" " * indent + f"{key}:")
+            pretty_print(value, indent + 2)
+    elif isinstance(data, list):
+        for item in data:
+            pretty_print(item, indent)
+    else:
+        print(" " * indent + str(data))

```

Comment:
there is a `pretty_print` in python:
```python
from pprint import pprint
```

Does it not do what you want?

Reply:
It's definitely good enough, I'll use it instead.

Reply:
It is used for dev only, I'll remove it.